optaplanner-examples/src/main/java/org/optaplanner/examples/vehiclerouting/optional/score/VehicleRoutingIncrementalScoreCalculator.java

```diff
@@ -125,71 +124,71 @@ public void afterEntityRemoved(Object entity) {
     }
 
     private void insertVehicle(Customer customer) {
-        Vehicle vehicle = customer.getVehicle();
-        if (vehicle != null) {
-            // Score constraint vehicleCapacity
-            int capacity = vehicle.getCapacity();
-            int oldDemand = vehicleDemandMap.get(vehicle);
-            int newDemand = oldDemand + customer.getDemand();
-            hardScore += Math.min(capacity - newDemand, 0) - Math.min(capacity - oldDemand, 0);
-            vehicleDemandMap.put(vehicle, newDemand);
-            if (customer.getNextCustomer() == null) {
-                // Score constraint distanceFromLastCustomerToDepot
-                softScore -= customer.getLocation().getDistanceTo(vehicle.getLocation());
-            }
-        }
+        // Vehicle vehicle = customer.getVehicle();
+        // if (vehicle != null) {
+        //     // Score constraint vehicleCapacity
+        //     int capacity = vehicle.getCapacity();
+        //     int oldDemand = vehicleDemandMap.get(vehicle);
+        //     int newDemand = oldDemand + customer.getDemand();
+        //     hardScore += Math.min(capacity - newDemand, 0) - Math.min(capacity - oldDemand, 0);
+        //     vehicleDemandMap.put(vehicle, newDemand);
+        //     if (customer.getNextCustomer() == null) {
+        //         // Score constraint distanceFromLastCustomerToDepot
+        //         softScore -= customer.getLocation().getDistanceTo(vehicle.getLocation());
+        //     }
+        // }
     }
 
     private void retractVehicle(Customer customer) {
-        Vehicle vehicle = customer.getVehicle();
-        if (vehicle != null) {
-            // Score constraint vehicleCapacity
-            int capacity = vehicle.getCapacity();
-            int oldDemand = vehicleDemandMap.get(vehicle);
-            int newDemand = oldDemand - customer.getDemand();
-            hardScore += Math.min(capacity - newDemand, 0) - Math.min(capacity - oldDemand, 0);
-            vehicleDemandMap.put(vehicle, newDemand);
-            if (customer.getNextCustomer() == null) {
-                // Score constraint distanceFromLastCustomerToDepot
-                softScore += customer.getLocation().getDistanceTo(vehicle.getLocation());
-            }
-        }
+        // Vehicle vehicle = customer.getVehicle();
+        // if (vehicle != null) {
+        //     // Score constraint vehicleCapacity
+        //     int capacity = vehicle.getCapacity();
+        //     int oldDemand = vehicleDemandMap.get(vehicle);
+        //     int newDemand = oldDemand - customer.getDemand();
+        //     hardScore += Math.min(capacity - newDemand, 0) - Math.min(capacity - oldDemand, 0);
+        //     vehicleDemandMap.put(vehicle, newDemand);
+        //     if (customer.getNextCustomer() == null) {
+        //         // Score constraint distanceFromLastCustomerToDepot
+        //         softScore += customer.getLocation().getDistanceTo(vehicle.getLocation());
+        //     }
+        // }
```

Comment:
Please don't do this. This is what Git is for.

Reply:
Of course. Draft quality level :slightly_smiling_face: